nf_core/components/components_command.py

```diff
@@ -158,22 +158,22 @@ def load_lint_config(self):
         except FileNotFoundError:
             log.debug(f"No lint config file found: {config_fn}")
 
-    def check_component_structure(self, component_name):
+    def check_modules_structure(self):
```

Comment:
Do we not need to also check the subworkflows directory?

Reply:
Answered the same question to @awgymer  😅 https://github.com/nf-core/tools/pull/2000/files#r1014251576